genesis.json

```diff
@@ -5,7 +5,7 @@
     "balance": "39097695890410958904",
     "stakingvalue": "0",
     "delegation": {
-      "address": "erd1qqqqqqqqqqqqqpgquade0dh30gquzgqj0s6yjuwwev3zav0wfmuspujwe5",
+      "address": "erd1qqqqqqqqqqqqqpgqxwakt2g7u9atsnr03gqcgmhcv38pt7mkd94q6shuwt",
```

Comment:
Address is correct for `owner = erd1ghg3vusfhy0wx92kvafw2kxhu2pdz8rrt3kr6yd3zapfwwwfd94qvnkddh` and `nonce = 0` and `VM Type = Arwen`.